javaparser-core/src/main/java/com/github/javaparser/utils/CollectionStrategy.java

```diff
@@ -24,9 +28,9 @@
             return parse(file.toFile()).getStorage()
                     .map(CompilationUnit.Storage::getSourceRoot);
         } catch (ParseProblemException e) {
-            Log.info("Problem parsing file %s", () -> file);
+            Log.info("Problem parsing file %s", () -> file, e);
```

Comment:
This is not log4j :-)

Reply:
Yes i see it too late :-( com.github.javaparser.utils.Log

Reply:
It is of course a weird thing. Here is the background if you're interested: https://javaparser.org/javaparsers-logging-framework-in-one-file/